src/index.js

```diff
@@ -1,64 +1,59 @@
-require("dotenv").config();
-const { Octokit } = require("@octokit/core");
-const axios = require("axios").default;
-const CronJob = require("cron").CronJob;
+require('dotenv').config()
+const { Octokit } = require('@octokit/core')
+const axios = require('axios').default
+const CronJob = require('cron').CronJob
 
-const TOKEN = process.env.TOKEN;
-const COLUMN_NODE_ID = process.env.COLUMN_NODE_ID;
-const NOTIFIER_URL = process.env.NOTIFIER_URL;
-const MENTION = process.env.MENTION;
-const PR_TIME = process.env.PR_TIME || "0 9,18 * * 1-5";
-const SOZVON_TIME = process.env.SOZVON_TIME || "0 21 * * 1-5";
+const TOKEN = process.env.TOKEN
+const COLUMN_NODE_ID = process.env.COLUMN_NODE_ID
+const NOTIFIER_URL = process.env.NOTIFIER_URL
+const MENTION = process.env.MENTION
+const PR_TIME = process.env.PR_TIME || '0 9,18 * * 1-5'
```

Comment:
comment this cron setting in simple words

Reply:
Done